scripts/rebase.sh

```diff
@@ -427,7 +443,7 @@ rebase_to() {
         title "## Updating vendor directory"
         go mod vendor
         title "## Patching vendor directory"
-        git apply scripts/rebase_patches/*
+        git apply scripts/rebase_patches/*.patch 2>/dev/null || true
```

Comment:
Why hide the errors from this step?

Reply:
Right, I'd keep those,  otherwise it can be hard to trace

Reply:
Soon having patches to apply will be the exception and I was bothered by CI highlighting the expected. But I agree and will instead apply conditionally on whether patches exist.

Reply:
done